### ISSUE TITLE
Rename "buffering of heat in heat pumps" strategy

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmoses#811]
![etmoses](https://cloud.githubusercontent.com/assets/1303760/13223335/205dcf9a-d984-11e5-91dd-e02d91021d0f.png)

**Comment by ChaelKruip** [comment]
I think it should become something like "Only buffer with heat pumps if network has spare capacity".

Do you agree @antw @dennisschoenmakers ?

**Comment by antw** [comment]
> Do you agree @antw?

No. :laughing: 

But only because the new wording doesn't accurately describe the strategy. The old one doesn't, either, so it *does* need a change.

With the strategy disabled, heat pumps do not buffer; they will run only as much as is needed satisfy heat demand in that frame (time step). With the strategy enabled, HPs may also create extra heat which is stored in the buffer (but not exceeding network capacity).

Either:

1. We should adopt your wording and change the strategy so that heat pumps always buffer; or
2. We should reword it to something like "Buffering of heat **using** heat pumps"

**Comment by ChaelKruip** [comment]
> We should adopt your wording and change the strategy so that heat pumps always buffer

@antw would that be a lot of work? With hindsight that would probably be even more realistic than the recent changes you made to the heat pump behaviour (https://github.com/quintel/etmoses/issues/503).  Wish I had realised this earlier :sob:

**Comment by antw** [comment]
> @antw would that be a lot of work?

Not at all.

**Comment by ChaelKruip** [comment]
> Not at all.

Than I would be in favour of option 1 :smile:
